rules-tests/TypeDeclaration/Rector/Class_/TypedPropertyFromCreateMockAssignRector/Fixture/some_fixture.php.inc

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Class_\TypedPropertyFromCreateMockAssignRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+use Rector\Tests\TypeDeclaration\Rector\Class_\TypedPropertyFromCreateMockAssignRector\Source\SomeMockedClass;
+
+class SomeTest extends TestCase
+{
+    public $someMock;
+
+    protected function setUp(): void
+    {
+        $this->someMock = $this->createMock(SomeMockedClass::class);
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Class_\TypedPropertyFromCreateMockAssignRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+use Rector\Tests\TypeDeclaration\Rector\Class_\TypedPropertyFromCreateMockAssignRector\Source\SomeMockedClass;
+
+class SomeTest extends TestCase
+{
+    public \PHPUnit\Framework\MockObject\MockObject $someMock;
```

Comment:
shouldn't this be the intersection type?

```suggestion
    public \PHPUnit\Framework\MockObject\MockObject&SomeMockedClass $someMock;
```

Reply:
That would be incorrect, as `MockObject` is generic type.
Also, goal of this rule is to add type as soon as PHP 7.1